tests/rancher-cis-benchmark/rancher_cis_benchmark.go

```diff
@@ -0,0 +1,697 @@
+			Name: "Set Values.global.cattle.systemDefaultRegistry",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.cattle.systemDefaultRegistry", "test-registry"),
+		},
+		{
+			Name: "Set Values.tolerations",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				Set("tolerations", testTolerations),
+		},
+		{
+			Name: "Set Values.nodeSelector",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				Set("nodeSelector", testNodeSelector),
+		},
+		{
+			Name: "Set .Values.global.cattle.psp.enabled to true",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				Set("global.cattle.psp.enabled", true),
+		},
+		{
+			Name: "Set .Values.global.cattle.psp.enabled to false",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				Set("global.cattle.psp.enabled", false),
+		},
+		{
+			Name: "Set Values.global.cattle.clusterName",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.cattle.clusterName", "test-cluster"),
+		},
+		{
+			Name: "Set .Values.global.kubectl.repository and .Values.global.kubectl.tag",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.kubectl.repository", "test-kubectl-repo").
+				SetValue("global.kubectl.tag", "v1.20.11"),
+		},
+		{
+			Name: "Set .Values.global.kubectl.imagePullPolicy to IfNotPresent",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.imagePullPolicy", "IfNotPresent"),
+		},
+		{
+			Name: "Set .Values.global.kubectl.imagePullPolicy to Always",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.imagePullPolicy", "Always"),
+		},
+		{
+			Name: "Set .Values.global.kubectl.imagePullPolicy to Never",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("global.imagePullPolicy", "Never"),
+		},
+		{
+			Name: "Set .Values.image.cisoperator.repository and .Values.image.cisoperator.tag",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("image.cisoperator.repository", "test/cis-operator").
+				SetValue("image.cisoperator.tag", "v1.0.0"),
+		},
+		{
+			Name: "Set securityScan and sonobuoy image values",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				SetValue("image.securityScan.repository", "test/security-scan").
+				SetValue("image.securityScan.tag", "v1.1.0").
+				SetValue("image.sonobuoy.repository", "test/sonobuoy").
+				SetValue("image.sonobuoy.tag", "v1.2.0"),
+		},
+		{
+			Name: "Set alerts configuration values",
+
+			TemplateOptions: chart.NewTemplateOptions(DefaultReleaseName, DefaultNamespace).
+				Set("alerts.enabled", true).
```

Comment:
This can be combined into one `.Set()` call

Reply:
Happens in a few more places.

Reply:
fixed in all places.